events/models.py

```diff
@@ -928,7 +928,7 @@ class Feedback(models.Model):
     name = models.CharField(verbose_name=_('Name'), max_length=255, blank=True)
     email = models.EmailField(verbose_name=_('E-mail'))
     subject = models.CharField(verbose_name=_('Subject'), max_length=255, blank=True)
-    body = models.TextField(verbose_name=_('Body'), blank=True)
+    body = models.TextField(verbose_name=_('Body'), max_length=10000, blank=True)
```

Comment:
This is 5th subject. Should this have a migration file?

Reply:
hei! thanks for spotting this one out. it got here by mistake indeed. this change has already been merged a while ago.